pkg/informer/informer.go

```diff
@@ -571,19 +618,6 @@ func gvrsToDiscoveryData(gvrs map[schema.GroupVersionResource]struct{}) []*metav
 	return discoveryData
 }
 
-// DiscoveryData implements resourcequota.NamespacedResourcesFunc and is intended to be used by the quota subsystem.
-func (d *DynamicDiscoverySharedInformerFactory) DiscoveryData() ([]*metav1.APIResourceList, error) {
```

Comment:
What happened to this?

Reply:
`DynamicDiscoverySharedInformerFactory` now implements (most of) `discovery.DiscoveryInterface`, so this ad-hoc method has been superseded by this interface methods.